openshift/templates/issuer-api/issuer-api-deploy.yaml

```diff
@@ -174,28 +178,28 @@ objects:
               env:
                 - name: PORT
                   value: ${SERVICE_PORT}
-                - name: DB_SERVICE
-                  value: ${DB_SERVICE}${DB_SUFFIX}
+                - name: DB_HOST
```

Comment:
This should not be changed, as the application expects the `DB_SERVICE` environment variable to be populated: https://github.com/bcgov/issuer-kit/blob/master/docker/docker-compose.yml#L12
You can chance the name of the parameter that will be interpolated to assign the value in the template, but not the value of the environment variable itself (unless you also update all of the issuer-kit code that relies on it, which we likely won't do 😉 )